src/views/home/components/transaction-actions/transaction-actions.view.tsx

```diff
@@ -0,0 +1,27 @@
+import { Link } from "react-router-dom";
+
+import { ReactComponent as Deposit } from "src/assets/icons/deposit.svg";
+import { ReactComponent as Withdraw } from "src/assets/icons/withdraw.svg";
+import useTransactionActionsStyles from "src/views/home/components/transaction-actions/transaction-actions.styles";
+
+const TransactionActions = () => {
+  const classes = useTransactionActionsStyles();
+  return (
+    <div className={classes.buttons}>
+      <Link to={"/"} className={classes.actionBtn}>
+        <p className={classes.circle}>
```

Comment:
Why the `circle` is a `<p>` element?

Reply:
because Link is an `a` element

Reply:
I know, but a <p> element is for paragraphs. Why you don't apply the circle styles to the `a` element and you keep just the icon inside?